unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAudioHandlerRemote.cs

```diff
@@ -62,13 +69,23 @@ private void Update()
         {
             if (footstepJump != null)
                 footstepJump.Play(true);
+            if (bodyPartReferenceHandler != null) {
+                if (stickersController != null)
+                    stickersController.PlaySticker("footstepJump", bodyPartReferenceHandler.footR.position, Vector3.up, false);
+            }
         }
 
         // Landed
-        if (blackBoard.isGrounded && !isGroundedPrevious)
-        {
+        if (blackBoard.isGrounded && !isGroundedPrevious) {
```

Comment:
Please leave a new line before brackets.
```suggestion
        if (blackBoard.isGrounded && !isGroundedPrevious) 
        {
```